types/react-big-calendar/index.d.ts

```diff
@@ -105,14 +114,14 @@ export interface HeaderProps {
 
 export interface Components {
     event?: React.SFC | React.Component | React.ComponentClass | JSX.Element;
-    eventWrapper?: React.SFC | React.Component | React.ComponentClass | JSX.Element;
+    eventWrapper?: React.SFC<EventWrapperProps> | React.Component<EventWrapperProps> | React.ComponentClass<EventWrapperProps> | JSX.Element;
```

Comment:
Do we loose some information with
```suggestion
    eventWrapper?: React.ReactType<EventWrapperProps>;
```
?

Reply:
Although I think we shouldn't be accepting string since this would cause warnings in dev because no intrinsic elements accepts props like `isRtl` or `accessors`.

Reply:
I'll change it to React.ComponentType like it is done for the header component. This type only allows a component class or function